### DISCUSSION TITLE
Curved NavBar - Request for help to complete it

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1321] [Ideas / feature requests]
This updated tailwind play bears refence: https://play.tailwindcss.com/kY5oJqQXQv

![CurvedNavBar](https://user-images.githubusercontent.com/5799306/200110856-bdf1b1a6-97b0-42b2-b3f6-024aed884649.gif)

The original, sadly incomplete project is here: https://github.com/anhsirk0/tailwind-navbar, thanks to @anhsirk0. He had posted this Daisy Show & Tell: https://github.com/saadeghi/daisyui/discussions/1158

Thing is, I don't know enough css / tailwind to complete this, I played around converting it to just plain from react and added some hovering.

One of the reasons I think daisyUI could use something like this is because its really awesome and is nowhere to be found. Is it possible to get help to get this complete and perhaps also make is smoother and logical.

I thank you humbly.

**Comment by anhsirk0** [comment]
I finished it you can check it out.
[Live version](https://anhsirk0-ts-projects.netlify.app/bottom-navbar)
[Github repo](https://github.com/anhsirk0/tailwind-navbar)
[typescript version](https://github.com/anhsirk0/ts-projects/blob/main/components/BottomNavbar/BottomNavbar.tsx)

Edit: This issue not daisyUI related, I haven't used daisyUI in these projects.

**Reply by Mashiane** [reply]
Wow, thank you so much. I will check it out asap.
